config/stage.json

```diff
@@ -55,7 +55,7 @@
 		"jetpack/features-section/jetpack": true,
 		"jetpack/features-section/simple": true,
 		"jetpack/magic-link-signup": true,
-		"jetpack/plugin-management": false,
+		"jetpack/plugin-management": true,
```

Comment:
Do we want to enable this for the staging environment? We will ship the changes to Calypso Blue few days after the launch date.